Material.Blazor/Components/Dialog/MBDialog.razor

```diff
@@ -1,49 +1,44 @@
 ﻿@namespace Material.Blazor
 @inherits ComponentFoundation
 
+<div @ref="DialogElem"
+     class="mdc-dialog @ActiveConditionalClasses @(@class)"
+     style="@style"
+     id="@id"
+     @attributes="@MyAttributes"
+     aria-modal="true">
 
-@if (IsOpen)
-{
-    <div @key="@Key"
```

Comment:
This is the only actual other change: no more key. When we re-open the same dialog, the child elements were already initialized correctly once, so no need to discard that entire dom tree.